x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_form.tsx

```diff
@@ -133,6 +134,7 @@ export const PipelineForm: React.FunctionComponent<PipelineFormProps> = ({
           hasVersion={Boolean(defaultValue.version)}
           isEditing={isEditing}
           canEditName={canEditName}
+          defaultValue={defaultValue}
```

Comment:
Can you explain why this wasn't needed before, but is now?

Reply:
Sure! I passed this field so I can obtain the `_meta` field from it when deciding whether it is set to a non-empty object (which decides whether we will display the JSON Editor _meta field).

However, now that I think about it, it might be more consistent to pass a `hasMeta` field instead as it is done for `hasVersion`. I will change this in the next commit, please let me know if that would be better.

Reply:
Ah, I see. Yeah, I think I might prefer to use `hasMeta`.